kong/constants.lua

```diff
@@ -70,6 +70,7 @@ return {
   DICTS = {
     "kong",
     "kong_cache",
+    "kong_rl_counters",
```

Comment:
I am sure you thought about it a lot, but this feature seems to make rate-limiting plugin a core feature, while it is still a plugin. It feels a bit bad that plugins inject stuff in core. About naming, I'd like `kong_rate_limiting` as `dict` name better as it pollutes core a bit less.

Reply:
> I am sure you thought about it a lot, but this feature seems to make rate-limiting plugin a core feature, while it is still a plugin. It feels a bit bad that plugins inject stuff in core.

Yep. This is also why the fix took so long to appear... We have little choice for now though.

 > I'd like kong_rate_limiting as dict name better as it pollutes core a bit less.

We can do that. Probably `kong_rate_limiting_counter` for completesness's sake then.